extension-manifest-v3/src/utils/engines.js

```diff
@@ -430,4 +432,11 @@ chrome.alarms.onAlarm.addListener((alarm) => {
   }
 });
 
+observe('experimentalFilters', (experimentalFilters) => {
```

Comment:
You are using this in line 42, but this observe is in 435. Please move it close to where the key is used.